code/game/objects/obj_defense.dm

```diff
@@ -224,7 +224,8 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 
 ///called after the obj takes damage and integrity is below integrity_failure level
 /obj/proc/obj_break(damage_flag)
-	return
+	SHOULD_CALL_PARENT(TRUE)
```

Comment:
does calling parent on all those subtypes cause any issues?

Reply:
All of them that I could tell had nothing or irrelevant code in their parents.